pandas/core/base.py

```diff
@@ -785,7 +784,7 @@ def array(self) -> ExtensionArray:
 
         return result
 
-    def to_numpy(self, dtype=None, copy=False):
+    def to_numpy(self, dtype=None, copy=False, na_value=lib._no_default, **kwargs):
```

Comment:
need kwargs?

Reply:
Yes, here the kwargs are needed, as this is passed through to the ExtensionArray implementation (this way, the EA can customize the conversion to numpy, in case it needs more options)

Reply:
like what? 

can we be explicit about what the keywords actually are?

Reply:
I think the point is to be arbitrary, so being explicit will be hard :)

I do have one in `DecimalArray.to_numpy(decimals=10)` to control the precision when converting. That's a bit contrived since you could also round before doing the conversion.